consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/validation/ProposalPayloadValidator.java

```diff
@@ -144,6 +144,10 @@ public boolean validate(final SignedData<ProposalPayload> signedPayload) {
   }
 
   private boolean validateBlock(final Block block) {
+    if (blockValidator == null) {
+      throw new RuntimeException("Block validation not possible, no block validator.");
```

Comment:
nit: this should never happen so perhaps throwing an IllegalStateException instead would be clearer

Reply:
Decided to replace with `checkState(...)` which results in an `IllegalStateException` if the predicate fails.